tests/Unit/UserTest.php

```diff
@@ -23,6 +23,13 @@ final class UserTest extends TestCase
     protected $namespace;
     protected $refresh = true;
 
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+
+        db_connect()->table('auth_identities')->truncate();
```

Comment:
Was this not already handled by `$refresh = true`?

Reply:
It can probably be removed again. I was hitting weird errors during tests and tried that to make sure, but I think I fixed it a different way.